_includes/graphql/customer-payment-tokens.md

```diff
@@ -0,0 +1,16 @@
+The `CustomerPaymentTokens` output object contains an array of `items`.
+
+Attribute | Data Type | Description
+--- | --- | ---
+`items` | [PaymentToken] | Contains an array of customer payment tokens
```

Comment:
What's the difference between a "customer payment token" and "customer payment method"? Payment method makes more sense to me. Can we use that throughout?

Reply:
I can't change the terminology. Tokens are managed here, not payment methods.

Reply:
@erikmarr , payment token and method are different. The payment token - tokenized on Payment Gateway credit card, payment method - an integration to provide a possibility to make a payment transaction, there are no terms like "customer payment method" because the payment method doesn't belong to any customer.